abodepy/devices/camera.py

```diff
@@ -115,7 +115,47 @@ def image_to_file(self, path, get_image=True):
 
         return True
 
+    def privacy_mode(self, enable):
+        """Set camera privacy mode (camera on/off)."""
+        if self._json_state['privacy']:
+            privacy = '0' if enable else '1'
+
+            url = CONST.PARAMS_URL + self.device_id
+
+            camera_data = {
+                'name': self.name,
+                'mac': self._json_state['camera_mac'],
+                'privacy': privacy,
+                'action': 'setParam',
+                'area': '1',
+                'no': '1',
+                'id': self.device_id
+            }
```

Comment:
Do you *need* to send all of this data, or can you get away with just the privacy and maybe action parameters? I don't think I need to send the ID, name, mac, area, etc on any other requests that explicitly target a device_id in the URL.

Reply:
From my testing, the `mac`, `privacy`, `action` and `id` are the required parameters. I don't think `area` will ever be a factor unless we get separate zone control in Abode. I'm unsure what `no` is for. I'll remove `name`, `area` and `no`.